<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1 (Python 3.10.6 on darwin)" generated="20230821 21:08:22.988" rpa="false" schemaversion="4">
<suite id="s1" name="Robot-Framework" source="/Users/kamil/Repositories/robot-framework">
<suite id="s1-s1" name="Tests" source="/Users/kamil/Repositories/robot-framework/Tests">
<suite id="s1-s1-s1" name="Website Example" source="/Users/kamil/Repositories/robot-framework/Tests/website_example">
<suite id="s1-s1-s1-s1" name="TestSuite" source="/Users/kamil/Repositories/robot-framework/Tests/website_example/TestSuite.robot">
<test id="s1-s1-s1-s1-t1" name="Login User with Password" line="7">
<kw name="Connect to Server" library="keywords">
<kw name="Connect">
<arg>fe80::aede:48ff:fe00:1122</arg>
<msg timestamp="20230821 21:08:23.020" level="FAIL">No keyword with name 'Connect' found.</msg>
<status status="FAIL" starttime="20230821 21:08:23.018" endtime="20230821 21:08:23.020"/>
</kw>
<status status="FAIL" starttime="20230821 21:08:23.017" endtime="20230821 21:08:23.021"/>
</kw>
<kw name="Login User" library="keywords">
<arg>ironman</arg>
<arg>1234567890</arg>
<status status="NOT RUN" starttime="20230821 21:08:23.022" endtime="20230821 21:08:23.023"/>
</kw>
<kw name="Verify Valid Login" library="keywords">
<arg>Tony Stark</arg>
<status status="NOT RUN" starttime="20230821 21:08:23.023" endtime="20230821 21:08:23.023"/>
</kw>
<kw name="Close Server Connection" library="keywords" type="TEARDOWN">
<kw name="Disconnect">
<msg timestamp="20230821 21:08:23.026" level="FAIL">No keyword with name 'Disconnect' found.</msg>
<status status="FAIL" starttime="20230821 21:08:23.025" endtime="20230821 21:08:23.027"/>
</kw>
<status status="FAIL" starttime="20230821 21:08:23.024" endtime="20230821 21:08:23.028">No keyword with name 'Disconnect' found.</status>
</kw>
<tag>positive</tag>
<status status="FAIL" starttime="20230821 21:08:23.016" endtime="20230821 21:08:23.028">No keyword with name 'Connect' found.

Also teardown failed:
No keyword with name 'Disconnect' found.</status>
</test>
<doc>A test suite for valid login</doc>
<status status="FAIL" starttime="20230821 21:08:23.010" endtime="20230821 21:08:23.028"/>
</suite>
<status status="FAIL" starttime="20230821 21:08:23.008" endtime="20230821 21:08:23.030"/>
</suite>
<status status="FAIL" starttime="20230821 21:08:23.006" endtime="20230821 21:08:23.031"/>
</suite>
<status status="FAIL" starttime="20230821 21:08:22.991" endtime="20230821 21:08:23.032"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
<stat pass="0" fail="1" skip="0">positive</stat>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Robot-Framework">Robot-Framework</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1" name="Tests">Robot-Framework.Tests</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1-s1" name="Website Example">Robot-Framework.Tests.Website Example</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1-s1-s1" name="TestSuite">Robot-Framework.Tests.Website Example.TestSuite</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20230821 21:08:23.014" level="ERROR">Error in file '/Users/kamil/Repositories/robot-framework/Tests/website_example/resources/keywords.resource' on line 4: Importing library '/Users/kamil/Repositories/robot-framework/Tests/website_example/resources/CustomLibrary.py' failed: ModuleNotFoundError: No module named 'TestObject'
Traceback (most recent call last):
  File "/Users/kamil/Repositories/robot-framework/Tests/website_example/resources/CustomLibrary.py", line 1, in &lt;module&gt;
    from TestObject import TestObject
PYTHONPATH:
  /Users/kamil/Repositories/robot-framework/Tests/website_example/resources
  /Users/kamil/Repositories/robot-framework/lib
  /Users/kamil/Repositories/robot-framework/resources
  /Users/kamil/.vscode/extensions/d-biehl.robotcode-0.51.1/bundled/libs
  /Users/kamil/.vscode/extensions/d-biehl.robotcode-0.51.1/bundled/tool/robotcode
  /opt/homebrew/Cellar/python@3.10/3.10.6_2/Frameworks/Python.framework/Versions/3.10/lib/python310.zip
  /opt/homebrew/Cellar/python@3.10/3.10.6_2/Frameworks/Python.framework/Versions/3.10/lib/python3.10
  /opt/homebrew/Cellar/python@3.10/3.10.6_2/Frameworks/Python.framework/Versions/3.10/lib/python3.10/lib-dynload
  /Users/kamil/Library/Python/3.10/lib/python/site-packages
  /opt/homebrew/lib/python3.10/site-packages</msg>
</errors>
</robot>
